slides/01-introduction.qmd

```diff
@@ -141,3 +141,116 @@ inference vs. prediction
 library(tidymodels)
 ```
 
+##  {background-image="images/tm-org.png" background-size="contain"}
+
+## The whole game
+
+```{r diagram-split, echo = FALSE}
+#| fig-align: "center"
+
+knitr::include_graphics("images/whole-game-split.jpg")
+```
+
+## The whole game
```

Comment:
We color coordinate based on data in a lot of places. The colors of the resamples and models are more similar to the test set color than the training set. 

Can you change that? Maybe use the colors that we have in the tidymodels book and in the later notes? 

I think that consistency is important here even for something subtle like the shapes and colors.

Reply:
I can definitely adapt the colors 👍 If aiming for the colors of the book/later notes: do you have the hex color code for those or would that mean dipping into OmniGraffle?

Reply:
The yellowish color for the training set is #ffe9bb and the blueish color for the test set is #e5e6ff

Reply:
Thanks! New colors in [73dd7b4](https://github.com/tidymodels/workshops/pull/137/commits/73dd7b44773143ccb0611c7af4d1239fb0fa0e75)